runtime/compiler/control/JITaaSCompilationThread.cpp

```diff
@@ -123,9 +124,9 @@ void handler_IProfiler_profilingSample(JITaaS::J9ClientStream *client, TR_J9VM *
    if (wholeMethodInfo)
       {
       // Serialize all the information related to this method
-      iProfiler->serializeAndSendIProfileInfoForMethod(method, comp, client);
+      iProfiler->serializeAndSendIProfileInfoForMethod(method, comp, client, abort);
```

Comment:
If you want `abort` to be returned, it's better to just use `return abort` rather than using a parameter passed by reference.